pkg/backends/fake/fake.go

```diff
@@ -5,28 +5,35 @@ import (
 
 	"go.uber.org/zap"
 	batchV1 "k8s.io/api/batch/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
-	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 
 	loadTestV1 "github.com/hellofresh/kangal/pkg/kubernetes/apis/loadtest/v1"
 )
 
 var (
-	sleepImage = "alpine"
-	imageTag   = "3.12.0"
+	imageName = "alpine"
+	imageTag  = "3.12.0"
 )
 
 // Fake enables the controller to run a LoadTest using Fake load provider which simulates load test
 type Fake struct {
+	backend    *Backend
 	kubeClient kubernetes.Interface
 	loadTest   *loadTestV1.LoadTest
 	logger     *zap.Logger
```

Comment:
Do you still need the Logger and KubeClient in here?

Reply:
nope, removed